app/index.js

```diff
@@ -0,0 +1,47 @@
+import angular from 'angular';
+import uiRouter from 'angular-ui-router';
+import ngMaterial from 'angular-material';
+import ngMessages from 'angular-messages';
+import componentsModule from './components';
+import pagesModule from './pages';
+import localStorage from 'angular-local-storage';
+import './style/sass/common.scss';
+
+const root = angular
+    .module('app', [
+        // 3rd party modules
+        uiRouter,
+        ngMaterial,
+        ngMessages,
+        localStorage,
+        // App modules
+        componentsModule,
+        pagesModule
+    ])
+    .constant("categories", "https://project-5613440220430148247.firebaseio.com/api/v1/categories.json")
```

Comment:
You should specify only base url of the project (that you will use in the future in making API calls), not an endpoint